### DISCUSSION TITLE
[Feat]: Code modification

### DISCUSSION THREAD

**Discussion by chenhuanguang** [coder/code-server#5666] [Q&A] [answered]
At present, I download and start the code server in the form of a package, but I want to modify some front-end codes. I don't know how to start

For example, what should I do if I want to delete the icon of this plug-in
<img width="633" alt="image" src="https://user-images.githubusercontent.com/56351544/196428599-57299b66-3f24-4777-b45d-9236c69c583e.png">

**Comment by jsjoeio** [comment, marked as answer]
Hmm..so you can right click and select "Hide 'Extensions'".

If you want to delete it from source, you'd have to create a patch, modify in `lib/vscode`, build from source and you'd have a new executable of code-server. Let me know if that makes sense!

**Reply by chenhuanguang** [reply]
Thanks, but I don't know how to do it. Can you tell me in detail

**Reply by jsjoeio** [reply]
No, but there are some docs [here](https://coder.com/docs/code-server/latest/CONTRIBUTING#patching-code). That should at least help you get started.
